recipes/r-schoolmath/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set version = '0.4.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-schoolmath
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/schoolmath_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/schoolmath/schoolmath_{{ version }}.tar.gz
+  sha256: 67cfb937f95c680d4bd3a274a43bff847342f4b962cbe421de537a16dd8d1064
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('schoolmath')"           # [not win]
+    - "\"%R%\" -e \"library('schoolmath')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=schoolmath
+  license: GPL-2.0-only
```

DESCRIPTION shows `GPL (>= 2)`, so

```suggestion
  license: GPL-2.0-or-later
```

---

Thanks for catching this, updated!